python/setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+  name = "do_username",
+  version = "1.0.2",
+  packages = ["do_username"],
+  license = "MIT",
```

Comment:
```suggestion
  license = "Apache-2.0",
```
All packages in this repo are licensed under Apache 2.0 -- please could you also add a copy of the LICENSE file from the other packages into this `python` directory?